src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  padding-right: 16px;
+  padding-left: 15px;
```

Comment:
Does it really have smaller padding from one side? This is weird

Reply:
yes